src/trellis/core/rendering/render.py

```
@@ -368,10 +372,24 @@ def _call_mount_hooks(session: RenderSession, element_id: str) -> None:
+                        await h()
+
+                task = asyncio.create_task(run_async_hook())
+                session._background_tasks.add(task)
+                task.add_done_callback(session._background_tasks.discard)
+            else:
+                # Sync hook: call directly
+                try:
+                    with callback_context(session, element_id):
+                        hook()
+                except Exception as e:
+                    logging.exception(f"Error in Stateful.on_mount: {e}")
```

Ruff (0.14.10)

Redundant exception object included in `logging.exception` call

(TRY401)